src/components/Organization/Dashboard/Invite.tsx

```diff
@@ -82,13 +92,13 @@ const OptionForm = () => {
                 padding={6}
                 borderTopRadius='xl'
                 cursor='pointer'
-                bg={field.value === 'admin' ? bgSecondary : 'transparent'}
+                bg={field.value === 'admin' ? radioBgSelect : 'transparent'}
```

Comment:
This `Flex` is almost identical to the other one (only difference is the border radius position). The `_dark` prop should be used here, but I'd wrap these two parts in a custom component, rather than repeating code as it's done now.